src/meta_tags.js

```diff
@@ -24,13 +24,15 @@ class MetaTags extends Component {
     }
   }
   extractChildren() {
+    if(!this.context.children) return;
```

Comment:
There will be no `context.children`. It should be props.children. Also destruct children from prop on top, so you don't have to write `this.props.children` twice.